roles/kubernetes/preinstall/defaults/main.yml

```diff
@@ -50,10 +50,6 @@ etc_hosts_localhost_entries:
       - localhost
       - localhost.localdomain
 
-# Minimal memory requirement in MB for safety checks
-minimal_node_memory_mb: 1024
-minimal_master_memory_mb: 1500
-
```

Comment:
Hi @VannTen 

Why is the minimal check of memory removed.  
Is it because kubeadm already has a check?
Ref to https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/constants/constants.go#L396

Reply:
Yes, that's exactly the point, as well as the kernel version check. Since kubeadm is already doing validation, (which we're not ignoring anymore :laughing:  ), let's not maintain our own.

Reply:
Isn't better to fail early than mid-way through the kubeadm install ? 

I would prefer to fail at this stage rather than to 10 min later

Reply:
Good point. I'd still like to defer to kubeadm, but I think we need a better story on download first, to have kubeadm available sufficiently early than we can use it for validation (assuming that we can run the preflight independantly, but I think that's possible).

I'll remove that commit for now.